### PR TITLE
Update ray release version 2.7.0rc0

### PR DESCRIPTION
**Pull request by GeneDer**
## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by zhe-thoughts — APPROVED** [review]

File: doc/source/ray-air/examples/dolly_lightning_fsdp_finetuning.ipynb [hunk 1]
```diff
@@ -76,7 +76,7 @@
        "    </tr>\n",
        "    <tr>\n",
        "        <td style=\"text-align: left\"><b>Ray version:</b></td>\n",
-       "        <td style=\"text-align: left\"><b>3.0.0.dev0</b></td>\n",
+       "        <td style=\"text-align: left\"><b>2.7.0rc0</b></td>\n",
```
Comment (aslonnie) [inline doc/source/ray-air/examples/dolly_lightning_fsdp_finetuning.ipynb:79]: why is this one changed from `3.0.0.dev0` shouldn't this be `2.7.0` on the 2.7.0 release branch

Comment (aslonnie) [reply doc/source/ray-air/examples/dolly_lightning_fsdp_finetuning.ipynb:79]: I guess this is cherrypicked in but with the wrong version number.

@matthewdeng or @GeneDer , could you fix this on the 2.7.0 branch?

Comment (GeneDer) [reply doc/source/ray-air/examples/dolly_lightning_fsdp_finetuning.ipynb:79]: makes sense, let me push a PR on 2.7.0 real quick

Comment (matthewdeng) [reply doc/source/ray-air/examples/dolly_lightning_fsdp_finetuning.ipynb:79]: We should probably just delete the output blocks instead. cc @woshiyyya

Comment (GeneDer) [reply doc/source/ray-air/examples/dolly_lightning_fsdp_finetuning.ipynb:79]: oh if that's the case I will let you guys figure it out XP. There are a few .ipynb files that need to clean up here

**Pull request merged**